### DISCUSSION TITLE
Multiple Scenarios in one input json?

### DISCUSSION THREAD

**Discussion by g-dixit** [NREL/REopt-API-Analysis#56] [8 - API: Application Programming Interface] [answered]
Can 2 separate Scenarios be submitted to REopt in the same input json file or they need to be run as separate optimizations? For example, running the optimization with 2 different utility rates for the same site.

**Comment by NLaws** [comment, marked as answer]
Each Scenario is a separate optimization/POST to the `job` endpoint.
